projects/ccd-case-ui-toolkit/src/lib/shared/pipes/complex/ccd-read-fields-filter.pipe.spec.ts

```diff
@@ -278,4 +323,25 @@ describe('ReadFieldsFilterPipe', () => {
     expect(RESULT[1].hidden).toEqual(true);
     expect(RESULT[2].hidden).toEqual(true);
   });
+  it('it shoulld evaluate showcondition and set the hidden property of field to true when value doesnt match within complex field even Formgroup passed with idPrefix passed as empty string', () => {
```

Comment:
@RiteshHMCTS - please remove the second "it" and correct typo to `should`. Also, "doesnt" should be `doesn\'t` (backslash escape needed). These corrections are also needed on 5 unit tests above this one

Reply:
@Daniel-Lam thanks for your review, addressed above comments